src/pages/TelaDeJogo.js

```diff
@@ -37,6 +37,40 @@ class TelaDeJogo extends Component {
     this.setState({ intervalId });
   }
 
+  savePoints({ target: { id } }) {
+    const { questions: { results } } = this.props;
+    const { difficulty } = results[0];
+
+    const difficultyPoints = () => {
+      const hardPoints = 3;
+      const mediumPoints = 2;
+      const easyPoints = 1;
+
+      switch (difficulty) {
+      case 'hard':
+        return hardPoints;
+
+      case 'medium':
+        return mediumPoints;
+
+      case 'easy':
+        return easyPoints;
+
+      default:
+        return 0;
+      }
+    };
+
+    if (id === 'correct') {
+      const magicMike = 10;
+      const time = 1;
+      const points = magicMike + (time * difficultyPoints());
+
+      const currentPoints = JSON.parse(localStorage.getItem('points'));
+      localStorage.setItem('points', JSON.stringify(points + currentPoints));
+    }
+  }
```

Comment:
```suggestion
  savePoints({ target: { id } }) {
    const { questions: { results } } = this.props;
    const { time } = this.state;
    const { difficulty } = results[0];

    const difficultyPoints = () => {
      const hardPoints = 3;
      const mediumPoints = 2;
      const easyPoints = 1;

      switch (difficulty) {
      case 'hard':
        return hardPoints;

      case 'medium':
        return mediumPoints;

      case 'easy':
        return easyPoints;

      default:
        return 0;
      }
    };

    if (id === 'correct') {
      const magicMike = 10;
      const points = magicMike + (time * difficultyPoints());

      const currentPoints = JSON.parse(localStorage.getItem('points'));
      localStorage.setItem('points', JSON.stringify(points + currentPoints));
    }
  }
```
Adicionado logica para armazenar o valor do tempo restante na questão